### DISCUSSION TITLE
PushName of a group Participant?

### DISCUSSION THREAD

**Discussion by alehsoares** [WhiskeySockets/Baileys#284] [General Questions]
How to get the PushName of a group Participant?

**Comment by joseorlandofp** [comment]
You mean get the pushName of a jid at any time or the pushName upon message upsert?

Even if the message is fired from a group the payload will still have message.pushName attr.

 ```js 
 socket.ev.on('messages.upsert', async (m) => {
    const message = m.messages[0];
    console.log(message);
    if (m.type !== 'notify') return;

   console.log(message.pushName);

  });
  ```

**Comment by alehsoares** [comment]
> How to get the PushName of a group Participant?
Hi!!

I'd like get the pushName of a jid at any time
